tasks/configure.yml

```diff
@@ -163,12 +165,26 @@
   debug: msg={{ ssh_key.stdout_lines }}
   when: archivematica_src_configure_dashboardsettings is defined
 
+- name: "Create archivematica user in AtoM server"
+  user:
+    name: "archivematica"
+    group: "users"
+    system: True
+    home: "/home/archivematica"
+    createhome: True
+    shell: "/bin/bash"
```

Comment:
Nice work

nitpick: although it is already a system user, should we set the shell to /bin/bash? does that break atom?